integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/IntegrationTestWatcher.java

```diff
@@ -211,8 +211,15 @@ public void interceptTestMethod(Invocation<Void> invocation,
   @Override
   public void handleTestExecutionException(ExtensionContext context, Throwable throwable)
       throws Throwable {
-    printHeader(String.format("Test failed %s()", methodName), "!");
-    getLogger().severe(getStackTraceAsString(throwable));
+    getLogger().info("Test Execution Exception is: " + throwable.getMessage());
```

Comment:
Hi Maggie, Instead of modifying the framework methods you can use condition check to skip a test. You can use the following annotation.